json/README.md

```diff
@@ -0,0 +1,15 @@
+JSON-java Codec
+===================
+
+This module adds support for encoding and decoding JSON via [JSON-java][].
```

Comment:
could you please add the link to https://www.json.org/json-en.html

Reply:
+